packages/legend-graph/src/graph/BasicModel.ts

```diff
@@ -99,6 +100,9 @@ const FORBIDDEN_EXTENSION_ELEMENT_CLASS = new Set([
  */
 export abstract class BasicModel {
   root: Package;
+
+  private _sdlcPointer: LegendSDLC | undefined;
```

Comment:
`private readonly`

Reply:
this is not read only. it is set during the build graph phase